orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/Executions.kt

```diff
@@ -85,7 +85,7 @@ fun Stage<out Execution<*>>.nextTask(task: Task) =
  * @throws IllegalArgumentException if there is no such task.
  */
 fun Stage<out Execution<*>>.task(taskId: String) =
-  getTasks().find { it.id == taskId } ?: throw IllegalArgumentException("No such task")
+  getTasks().find { it.id == taskId }
```

Comment:
Just by removing this I'm then forced by the compiler to handle `null` everywhere this method is called. Kotlin is awesome.